fido/fido.py

```diff
@@ -253,10 +255,23 @@ def get_agent(reactor, connect_timeout=None):
 
     # TODO: Would be nice to have https_proxy support too.
     http_proxy = os.environ.get('http_proxy')
+
+    tcp_nodelay = os.environ.get('tcp_nodelay')
```

Comment:
ack. I didn't like the idea of changing the function signature, but I guess passing it in as an optional parameter should solve the problem.